lib/surrealist/copier.rb

```diff
@@ -59,6 +79,24 @@ def wrap_schema_into_root(schema:, klass:, carrier:)
         result
       end
 
+      # Wraps schema into the specified root key if `root` is passed as not nil to Surrealist.
+      #
+      # @param [Hash] schema schema hash.
+      # @param [Object] carrier instance of Carrier class that carries arguments passed to +surrealize+
+      #
+      # @return [Hash] a hash with schema wrapped inside the specified root key.
+      def wrap_schema_into_specified_root(schema:, carrier:)
+        root_key = if carrier.camelize
```

Comment:
This looks very similar to `#wrap_schema_into_root`. Maybe we can DRY it out a bit? 🤔